src/Microsoft.NetCore.Analyzers/UnitTests/Security/DoNotDisableHeaderCheckingTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Test.Utilities;
+using Xunit;
+
+namespace Microsoft.NetCore.Analyzers.Security.UnitTests
+{
+    public class DoNotDisableHeaderCheckingTests : DiagnosticAnalyzerTestBase
+    {
+        [Fact]
+        public void TestLiteralDiagnostic()
+        {
+            VerifyCSharp(@"
+using System;
+using System.Web.Configuration;
+
+class TestClass
+{
+    public void TestMethod()
+    {
+        var httpRuntimeSection = new HttpRuntimeSection();
```

Comment:
Can you also add a test for `new HttpRuntimeSection() { EnableHeaderChecking = false };`? I don't think we generate an `ISimpleAssignmentOperation` for field/property initialization in object initializers.